packages/sliderInput/README.md

```diff
@@ -0,0 +1,3 @@
+# SliderInput
```

Comment:
🔴 i'd totally vote against naming this a slider-input: https://duckduckgo.com/?q=slider+input&va=b&t=hc&iax=images&ia=images

how about "slide toggle" or such?